### PR TITLE
[fix](invert index) fix the keyword type index length limit

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

1.  keyword index length limit 16383 Byte

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

File: be/src/olap/rowset/segment_v2/column_writer.cpp [hunk 1]
```diff
@@ -510,7 +510,7 @@ Status ScalarColumnWriter::append_nulls(size_t num_rows) {
         _bitmap_index_builder->add_nulls(num_rows);
     }
     if (_opts.inverted_index) {
-        _inverted_index_builder->add_nulls(num_rows);
+        RETURN_IF_ERROR(_inverted_index_builder->add_nulls(num_rows));
     }
     if (_opts.need_bloom_filter) {
         _bloom_filter_index_builder->add_nulls(num_rows);
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/column_writer.cpp:516]: add RETURN_IF_ERROR to other index branch too.

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.79 seconds
 stream load tsv:          533 seconds loaded 74807831229 Bytes, about 133 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162125430 Bytes

**Comment by zzzxl1993** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.05 seconds
 stream load tsv:          532 seconds loaded 74807831229 Bytes, about 134 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.5 seconds inserted 10000000 Rows, about 338K ops/s
 storage size: 17161883865 Bytes

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**